### DISCUSSION TITLE
Can I start the WEB and RTSP SERVER by CLI commands?

### DISCUSSION THREAD

**Discussion by gzm001** [alienatedsec/yi-hack-v5#402] [Q&A]
After flashing my yi demo 1080 according to the instructions, I cannot access the WEB but can access the camera via telnet. 
Can I start the WEB and RTSP SERVER by CLI commands?

**Comment by alienatedsec** [comment]
If you can access via telnet, it means that your `yi-hack-v5` folder is not on your SD card.

**Reply by gzm001** [reply]
I had checked ,be sure the floder "yi-hack-v5" is on the SD card.
So, what can I do ?


![1111](https://github.com/user-attachments/assets/7a1aca98-ef62-4098-b0b0-2c79dda378cf)

**Reply by alienatedsec** [reply]
what is the content of that folder?

**Reply by gzm001** [reply]
On the cam os, I had checked the floder, and found some info ... as this picture:
![111](https://github.com/user-attachments/assets/75093008-8228-4c74-beab-a6723a68ae95)

Then try to launch.sh
![222](https://github.com/user-attachments/assets/a5b36208-537c-4161-89d9-7afbb1c692c2)
, report as  this pic:

**Reply by gzm001** [reply]
BTW, I had check /tmp/sd floder, There were no file.

**Comment by gzm001** [comment]
he issue has been resolved. It was my SD card that was faulty, and it's working normally after replacement.
Thank you for your selfless help.
